### PR TITLE
026 add UI branding

### PR DESCRIPTION
**Pull request by miagreen23**

### CODE REVIEW

File: public/.DS_Store
Comment (glenn-hamilton) [inline public/.DS_Store:1]: 😢

File: public/assets/academy1.jpeg
Comment (glenn-hamilton) [inline public/assets/academy1.jpeg:1]: This file could do with a bit of compression

File: public/assets/academy2.jpeg
Comment (glenn-hamilton) [inline public/assets/academy2.jpeg:1]: This file could do with a bit of compression

File: public/kainoos.png
Comment (glenn-hamilton) [inline public/kainoos.png:1]: Is this file needed any more?

File: views/templates/components/notification-bar.ejs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="alert alert-warning" style="text-align: center;" role="alert">
+    This is a dark alert—check it out!
+
+            <!-- {% if errormessage %}
```
Comment (glenn-hamilton) [inline views/templates/components/notification-bar.ejs]: Is this needed?

File: .DS_Store
Comment (glenn-hamilton) [inline .DS_Store:1]: 😢

File: .github/workflows/weekday-evening-node-audit.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: "NPM Audit"
 
 on:
   schedule:
-    - cron: "30 14 * * 1-5"
+    - cron: "22 0 * * 1-5"
```
Comment (glenn-hamilton) [inline .github/workflows/weekday-evening-node-audit.yml]: This should be `0 22 * * 1-5`, as currently it'll run at `00:22`

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "start": "npx tsc && cp -R views dist && node dist/app.js",
+    "start": "npx tsc && cp -R views dist && cp -R public dist && nodemon dist/app.js",
```
Comment (glenn-hamilton) [inline package.json]: I'm not sure how `nodemon` will run in `AWS`. Can you make this not you `nodemon`?

You can create a separate `start-nodemon` script if you want to use `nodemon`

File: package.json [hunk 4]
```diff
@@ -23,10 +23,12 @@
   "homepage": "https://github.com/dcorry586/the-it-crowd-ui#readme",
   "dependencies": {
     "axios": "^1.6.0",
+    "bootstrap-icons": "^1.11.1",
     "chai": "^4.3.10",
     "ejs": "^3.1.9",
     "express": "^4.18.2",
     "mocha": "^10.2.0",
+    "nodemon": "^3.0.1",
```
Comment (glenn-hamilton) [inline package.json]: `nodemon` should be in `devDependencies`

**Review by siyandlovu1 — APPROVED** [review]

File: views/pages/home.ejs [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```
Comment (glenn-hamilton) [inline views/pages/home.ejs:1]: Could this be split up into reusable components? So that you don't have to include the `<head>` in every html page you build etc

File: views/templates/components/common-header.ejs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (glenn-hamilton) [inline views/templates/components/common-header.ejs]: Is this and `views/templates/components/common-nav.ejs` not the same thing?

File: views/templates/components/footer.ejs [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<footer>
```
Comment (glenn-hamilton) [inline views/templates/components/footer.ejs]: Is this used any more?

File: public/menu-icon.png
Comment (TaylorJCrawford) [inline public/menu-icon.png:1]: Don't think we need this anymore? -> I can update mine to use the one in your assets folder.

File: package-lock.json
Comment (TaylorJCrawford) [inline package-lock.json:1]: Don't need this file. :)

Comment (glenn-hamilton) [reply package-lock.json:1]: `bootstrap-icons` has been added, so best to keep this file committed to keep everything in sync

Comment (TaylorJCrawford) [reply package-lock.json:1]: No worries at all. Happy to close :)

**Review by glenn-hamilton — APPROVED** [review]
